babel.config.js

```diff
@@ -1,7 +1,6 @@
-module.exports = {
-  retainLines: true,
-  compact: true,
-  comments: false,
-  presets: ["module:metro-react-native-babel-preset"],
-  sourceMaps: false
+module.exports = function(api) {
+  api.cache(true);
```

Comment:
Could you wrap this line in an if statement checking if `api` is defined? it might be an issue in some cases (see https://github.com/FaridSafi/react-native-gifted-chat/issues/1423)

Reply:
```suggestion
  if (!!api) {
    api.cache(true);
  }
```